Makefile

```
@@ -898,6 +898,17 @@ PACK = \
 			fi; \
 		done; \
 	done; \
+	if ! grep -q "darwin" <<< "$(MEMO_TARGET)"; then \
+		if ! [ "$(1)" = "libiosexec1" ] || ! [ "$(1)" = "libiosexec-dev" ]; then \
+			if find $(BUILD_DIST)/$(1) | xargs file -ib | grep 'x-mach-binary' | head -1 | grep -q 'x-mach-binary'; then \
```

Couldn't this be simplified to
```suggestion
			if find $(BUILD_DIST)/$(1) | xargs file -ib | grep -q 'x-mach-binary'; then \
```
Unless there was a reason for having the extra `head` and `grep` that I missed.

---

It could, but it won't stop on the first Mach-O if changed to this, and will take more time as a result.